pkg/wookie/wookie.go

```diff
@@ -17,36 +17,41 @@ package wookie
 import (
 	"context"
 	"net/http"
+
+	"github.com/pkg/errors"
 )
 
-const HeaderName = "Warrant-Token"
+const WarrantTokenHeaderName = "Warrant-Token"
 const Latest = "latest"
 
-type wookieCtxKey struct{}
+type WookieCtxKey struct{}
```

Comment:
Why does this need to be exported out of the package?

Reply:
Not longer exported, added `GetWookieFromRequestContext` to fetch the value from context instead